warp10/src/main/java/io/warp10/script/formatted/FormattedWarpScriptFunction.java

```diff
@@ -71,11 +71,31 @@ public ArgumentsBuilder addArgument(Class<?> clazz, String name, String doc) {
       return this;
     }
 
+    public ArgumentsBuilder addListArgument(Class<?> subClazz, String name, String doc) {
```

Comment:
This change should be in a separate PR as it is not solely linked to the SHAPE/tensor functions